src/RA_Dlg_AchEditor.cpp

```diff
@@ -1079,7 +1078,7 @@ INT_PTR Dlg_AchievementEditor::AchievementEditorProc(HWND hDlg, UINT uMsg, WPARA
                                 m_ConditionClipboard.Add(NewCondition);
                             }
 
-                            (void)m_ConditionClipboard.Count();
+                            static_cast<void>(m_ConditionClipboard.Count());
```

Comment:
The `_NODISCARD` is doing it's job here. `Count()` returns a member variable, there's no reason it needs to be called. Suggest just removing it.

Reply:
done.